DEHPEcosimPro/ViewModel/Rows/ParameterDifferenceRowViewModel.cs

```diff
@@ -0,0 +1,156 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ParameterDifferenceRowViewModel.cs" company="RHEA System S.A.">
+//    Copyright (c) 2020-2021 RHEA System S.A.
+// 
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Nathanael Smiechowski, Arielle Petit.
+// 
+//    This file is part of DEHPEcosimPro
+// 
+//    The DEHPEcosimPro is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Lesser General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or (at your option) any later version.
+// 
+//    The DEHPEcosimPro is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Lesser General Public License for more details.
+// 
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with this program; if not, write to the Free Software Foundation,
+//    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace DEHPEcosimPro.ViewModel.Rows
+{
+    using CDP4Common.EngineeringModelData;
+
+    using ReactiveUI;
+
+    /// <summary>
+    /// Object to display on MainWindow, Value Diff
+    /// </summary>
+    public class ParameterDifferenceRowViewModel : ReactiveObject
+    {
+        /// <summary>
+        /// The Thing already on the data hub
+        /// </summary>
+        private Parameter oldThing;
+
+        /// <summary>
+        /// The Thing already on the data hub
+        /// </summary>
+        public Parameter OldThing
```

Comment:
if this is not a reactive property it does not need a backing field. Can just be a `{ get; set; }`